packages/api/src/parse-cli.ts

```diff
@@ -242,6 +242,12 @@ export default function parseCli(argv?: string | readonly string[]) {
         default: false,
         type: "boolean",
       },
+      "half-region-orchestrators-untrusted": {
+        describe:
+          "mark half of the orchestrators returned by /api/region as untrusted. For use in staging!",
+        default: false,
+        type: "boolean",
+      },
```

Comment:
Maybe we don't even need to merge this and thus don't need the flag. We can just change the image used for staging deployments on `livepeer-infra` to be the one coming from this branch.

Reply:
I did it initially, but then it was overwritten in today's deploy 😢 
But yah, let's try it again

Reply:
Oh I see. We could merge to `master` the change on `livepeer-infra` values/image for the staging deploy. The flag/env setting would also be overwritten if we didn't merge it anyway

Reply:
For now, I rebased this on top of `master` if you don't mind, which is OK as well for me. Just keep in mind that you might need a `reset` or `pull --rebase` to update to the `origin` version of the branch now

Reply:
Hey @victorges, I'm picking this up again - is reinstating the flag and passing that in for Staging still the way you want to go?